samcli/local/docker/container.py

```diff
@@ -53,7 +54,7 @@ def __init__(self,
         self._image = image
         self._cmd = cmd
         self._working_dir = working_dir
-        self._host_dir = host_dir
+        self._host_dir = posix_path(host_dir)
```

Comment:
Can we set the posix_path on all the volumes that get mounted? Apply to the `kwargs` on the `create()` method. 

When we mount other volumes in the future, this will just transparently work.